docs/changelog.rst

```diff
@@ -14,6 +14,10 @@ The full release history can be viewed `at the GitHub yank releases page <https:
 
 Bugfixes
 ^^^^^^^^
+- (PR `#830 <https://github.com/choderalab/perses/pull/830>`_)
+  Added limited support for resuming simulations from the CLI. 
+  Assumes simulations are only going to be resumed from the production step and not equilibration step.
+  ``LOGLEVEL`` can now be set with an environmental variable when using the CLI tool.
```

Comment:
Maybe we can say something about extending the simulation `n_cycles` now? I think that's a good thing to show.

Reply:
Good point!